BUILD

```diff
@@ -0,0 +1,131 @@
+repo = ["WORKSPACE",
+         ".bazelrc",
+         "//scala:sources",
+         "//scala/support:sources",
+         "//src/java/com/google/devtools/build/lib:sources",
+         "//src/java/io/bazel/rulesscala/io_utils:sources",
+         "//src/java/io/bazel/rulesscala/jar:sources",
+         "//src/java/io/bazel/rulesscala/scalac:sources",
+         "//src/java/io/bazel/rulesscala/worker:sources",
+         "//src/scala:sources",
+         "//src/scala/io/bazel/rules_scala/scrooge_support:sources",
+         "//src/scala/io/bazel/rules_scala/tut_support:sources",
+         "//src/scala/scripts:sources",
+         "//test:sources",
+         "//test/data:sources",
+         "//test/src/main/resources/scala/test:sources",
+         "//test/src/main/scala/scala/test/srcjars:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2/thrift3:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2/thrift4:sources",
+         "//test/tut:sources",
+         "//test_expect_failure/disappearing_class:sources",
+         "//test_expect_failure/scala_library_suite:sources",
+         "//test_expect_failure/transitive/java_to_scala:sources",
+         "//test_expect_failure/transitive/scala_to_java:sources",
+         "//test_expect_failure/transitive/scala_to_scala:sources",
+         "//thrift:sources",
+         "//tut_rule:sources",
+         "//twitter_scrooge:sources"]
+
+additions = [] 
+
+sh_test(
+    name = "build_targets_under_tests_package",
+    size = "large",
+    data = repo,
```

Comment:
So let me rephrase, you need:
  1. to set --javabase and --host_javabase like it is done in testenv in Bazel
  2. You will need to have "bazel" as a dependency to run inside the sandbox. Might not be important if it's on the PATH and installed outside of the workspace but I think it will fails on ci.bazel.io.